docs/adr/0019-keep-study-as-a-dto.md

```diff
@@ -0,0 +1,32 @@
+# Keep study as a DTO
+
+## Context and Problem Statement
+
+The study holds query and library entries that could be replaced respectively with complex query and fetcher instances.
+This poses the question: should the study remain a pure DTO object or should it contain direct object instances?
+
+## Considered Options
+
+* Keep study as DTO and use transformers
+* Replace entries with instances
+
+## Decision Outcome
+
+Chosen option: "Keep study as DTO and use transformators", because comes out best (see below).
+
+## Pros and Cons of the Options
+
+### Keep study as DTO and use transformators
+
+* Good, because makes GUI implementation probably easier down the line
+* Good, because no need for custom serialization
+* Good, because deactivated fetchers can be documented (important for traceable Searching (SLRs))
+* Bad, because Entries for databases and queries needed
+
+### Replace entries with instances
+
+* Good, because no need for database and query entries
+* Bad, because custom de-/serializers for fetchers and complex queries needed
```

Comment:
That depends a bit on how complicated the custom serialization is. Since in the end you only need a map Fetcher <-> Name is needed, and such a conversion already exists.

Reply:
The problem with this is that we need to keep the set of deactivated databases somewhere in memory to serialize them again.
In the future, we can replace this with records so I don't think this replacement is necessary now?
Regarding the complexity de-/serialization I'm not sure myself.

Reply:
> deactivated databases

That was actually something I was wondering about: why do you need to deactivate a fetcher (instead of simply removing it from the list)?

Reply:
The idea behind this is to document that certain databases/libraries are actively not searched instead for example just disregarded/forgotten.
This is just another aspect to make the search traceable.

Reply:
Ok, then we leave it as it's right now using custom Query/LibareryEntrys! 

Btw, you don't need to write an ADR for every implementation detail. Here it would have be perfectly fine to just add a comment outlining the advantages you see.

Reply:
I am aiming for creating a larger body of in-the-wirld ADRs to foster research on them. The example at hand can perfectly used to discuss the granularity of ADRs. - Thus, my wish is that the ADR is kept.